gfx_scene.h

```diff
@@ -311,6 +311,15 @@ enum GfxMaterialFlag
 };
 typedef uint32_t GfxMaterialFlags;
 
+enum GfxMaterialAlphaMode
+{
+    GfxMaterialAlphaMode_Opaque = 0,
+    GfxMaterialAlphaMode_Blend = 1,
+    GfxMaterialAlphaMode_Mask = 2,
+
+    GfxMaterialAlphaMode_Count
+};
```

Comment:
GLTF exposes 3 alpha modes: opaque, blend and clipped. Clipped also has an optional clip value (i.e. any alpha below some pre-sepcified value is considered clip) but we are currently just assuming the GLTF specs default of 0.5.